web_dashboard_tile/tile.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+       <record model="ir.ui.view" id="dashboard_tile_tree_view">
+            <field name="name">tile.tree</field>
+            <field name="model">tile</field>
+            <field name="type">tree</field>
+            <field name="arch" type="xml">
+                <tree string="Dashboard tiles">
+                    <field name="name"/>
+                    <field name="domain"/>
+                    <field name="model_id"/>
+                    <field name="action_id"/>
+                    <field name="count"/>
+                </tree>
+            </field>
+        </record>
+        
+        <record model="ir.ui.view" id="dashboard_tile_form_view">
+            <field name="name">tile.form</field>
+            <field name="model">tile</field>
+            <field name="type">form</field>
+            <field name="arch" type="xml">
+                <form string="Dashboard tiles" version="7.0">
+                    <group>
+                        <field name="name"/>
+                        <field name="domain"/>
+                        <field name="model_id"/>
+                        <field name="user_id"/>
+                        <field name="action_id"/>
+                        <!-- field name="count"/> -->
+                    </group>
+                </form>
+            </field>
+        </record>
+
+        <!-- CRM Lead Kanban View  -->
+       <record model="ir.ui.view" id="dashboard_tile_kanban_view">
+            <field name="name">tile.kanban</field>
+            <field name="model">tile</field>
+            <field name="type">kanban</field>
+            <field name="arch" type="xml">
+                <kanban edit="false" create="false">
+                    <field name="name"/>
+                    <field name="domain"/>
+                    <field name="model_id"/>
+                    <field name="action_id"/>
+                    <field name="count"/>
+                    <field name="color"/>
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div t-attf-class="oe_tile_color_#{kanban_getcolor(record.color.raw_value)} oe_dashbaord_tile oe_kanban_global_click" >
+                                <div class="oe_dropdown_toggle oe_dropdown_kanban">
+                                    <span class="oe_e">í</span>
+                                    <ul class="oe_dropdown_menu">
+                                        <t t-if="widget.view.is_action_enabled('edit')"><li><a type="edit">Edit...</a></li></t>
+                                        <t t-if="widget.view.is_action_enabled('delete')"><li><a type="delete">Delete</a></li></t>
+                                        <li><ul class="oe_kanban_colorpicker" data-field="color"/></li>
```

Comment:
:question: 
This 3 lines doesn't display menu item to edit / delete the 'tile' and change colors.
(Used with odoo 7.0 et tested on Firefox 35.0 & chromium.)
Do you have the same problem ?
Regards.